### PR TITLE
Db trips

### PR DESCRIPTION
**Pull request by Joanartdacosta**
loading info to trips section:
readapt data according new collections - db mongodb

### CODE REVIEW

File: app/components/common/menu_id/general_info/InfoBullets.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ import Bullet from "@/components/common/icons/bullets/Bullet";
 import TAB_BUTTONS_LABELS from "@/components/enums/tab_buttons_labels";
 
 export default function InfoBullets(props) {
+  const ARRAY = [];
```
Comment (doublethink13) [inline app/components/common/menu_id/general_info/InfoBullets.js:5]: this should be names BULLETS

File: app/components/common/menu_id/general_info/Introduction.js [hunk 2]
```diff
@@ -8,6 +8,8 @@ export default function Introduction(props) {
     current = useSelector((state) => state.tour.selectedTour);
   }
 
+  console.log(current.description1);
```
Comment (doublethink13) [inline app/components/common/menu_id/general_info/Introduction.js:11]: remove

File: app/components/common/menu_id/location/InfoPoints.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import Bullet from "@/components/common/icons/bullets/Bullet";
+
+export default function InfoPoints(props) {
+  const ARRAY = [];
```
Comment (doublethink13) [inline app/components/common/menu_id/location/InfoPoints.js:4]: BULLETS

File: app/helpers/api-util-trip.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
+// import clientPromise from "@/components/mongo/client";
```
Comment (doublethink13) [inline app/helpers/api-util-trip.js:1]: why commented out?

File: app/components/common/menu_id/general_info/Introduction.js (hunk 2 above)
Comment (Joanartdacosta) [reply app/components/common/menu_id/general_info/Introduction.js:11]: done

File: app/components/common/menu_id/general_info/InfoBullets.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/components/common/menu_id/general_info/InfoBullets.js:5]: done

File: app/components/common/menu_id/location/InfoPoints.js (hunk 3 above)
Comment (Joanartdacosta) [reply app/components/common/menu_id/location/InfoPoints.js:4]: correction done!

**Pull request merged**